### PR TITLE
fpm usage

### PR DESCRIPTION
**Pull request by booyaa**
i'd prefer to include an example nginx.conf, but i know that's off topic.

### CODE REVIEW

**Comment by yosifkit** [comment]
I think a snippet of an nginx.conf with the relevant parts could be very useful for users.  What do you think @tianon?

File: wordpress/content.md [hunk 1]
```diff
@@ -31,6 +31,10 @@ If you'd like to use an external database instead of a linked `mysql` container,
 	docker run --name some-wordpress -e WORDPRESS_DB_HOST=10.1.2.3:3306 \
 	    -e WORDPRESS_DB_USER=... -e WORDPRESS_DB_PASSWORD=... -d wordpress
 
+To get wordpress working with nginx, will require a custom build of nginx to enable fpm support. You'll also need to expose port 9000 (default fpm port for the wordpress build):
```
Comment (tianon) [inline wordpress/content.md]: "require a custom build of nginx to enable fpm support" has me a little confused.  I thought FastCGI support in nginx was well-baked and it was just a matter of configuring it appropriately?

@md5 you got this connection working, didn't you? (`nginx` + `wordpress:fpm`)

Comment (md5) [inline wordpress/content.md]: Yep: https://gist.github.com/md5/d9206eacb5a0ff5d6be0

**Comment by yosifkit** [comment]
Closing old PR; feel free to comment if you feel this was closed in error.

**Pull request closed without merging**